tests/unit/lib/WUI/nhttp/time_mock.cpp

```diff
@@ -15,4 +15,8 @@ extern "C" {
 uint32_t ticks_s() {
     return mock_time;
 }
+
+uint32_t ticks_ms() {
+    return mock_time;
+}
```

Comment:
It's kind-of suspicious that the `ticks_ms` and `ticks_s` would return the same thing. At least multiply 1000?